### ISSUE TITLE
MASM arithmetic instructions should accept constants

### ISSUE THREAD

**Issue by hackaugusto** [0xPolygonMiden/miden-vm#1299]
The following code snippet doesn't assemble:

```
const.V=2

begin
    push.1 mul.V
    push.1 div.V
    push.1 sub.V
    push.1 exp.V
end
```

It fails with:

```
malformed instruction `mul.V`: parameter 'V' is invalid
```

All the above instructions should accept a constants.

Motivation:

I want to implement a bitshift to the right with a division, and to make the code more readable I want to write:

```
const.SHIFT_RIGHT_8=256  # dividing by this constants is equivalent to a >>8

begin
    push.1 div.SHIFT_RIGHT_8
end

```

**Comment by hackaugusto** [comment]
Note: I'm aware the transformation from `>>8` to `/256` doesn't work for all situations. Nevertheless, for the cases it does work, the constants is best for readability.

**Comment by bitwalker** [comment]
This is implemented in #1277. Just making a note here so I revisit this issue once that's merged.

**Comment by hackaugusto** [comment]
@bitwalker maybe this can be closed?

**Comment by bobbinth** [comment]
@bitwalker this is already supported, right?

**Comment by bitwalker** [comment]
Yes, as mentioned in the other issue for `eq`, I believe I added support for constant operands to most instructions already, but if there are any missing it's easy to enable the functionality for them now.

**Comment by bobbinth** [comment]
Verified that this is now working. Closing.
